Persona3FES/Modules/Common/Functions.json

```diff
@@ -981,7 +981,7 @@
     {
         "Index": "0x0042",
         "ReturnType": "int",
-        "Name": "PARTY_CHECK",
+        "Name": "AI_CHK_PLAYER_ID",
```

Comment:
Perhaps we could investigate which ID corresponds to which party member so we could create an enum.

Reply:
Not important though